ci_framework/roles/edpm_build_images/tasks/package.yml

```diff
@@ -10,6 +10,9 @@
     cmd: >-
       buildah bud -f ./Containerfile.image
       -t edpm-hardened-uefi:{{ cifmw_edpm_build_images_tag }}
+      {% if ansible_distribution | lower == 'rhel' %}
```

Comment:
why would we not allow overriding for all cases - not just rhel.  Sometimes we have to use a diff centos version - so can't we just default {{ cifmw_edpm_build_images_base_image }} and always keep this clause?